internal/txcommon/contract_inputs.go

```diff
@@ -39,7 +40,9 @@ type BlockchainInvokeData struct {
 func RetrieveBlockchainInvokeInputs(ctx context.Context, op *core.Operation) (*core.ContractCallRequest, error) {
 	var req core.ContractCallRequest
 	s := op.Input.String()
-	if err := json.Unmarshal([]byte(s), &req); err != nil {
+	d := json.NewDecoder(bytes.NewReader([]byte(s)))
```

Comment:
Chatted that in the future we could have an `ffjson` in FireFly Common that handles this and we can use it in all place where we unmarshall/decode some payload